From f1c5b25e153c0c2a335632895a85000266dd475a Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Fri, 25 Jul 2008 23:06:02 +0100
Subject: [PATCH] Do not send low level debugging to the modem

Hey,

I see no reason to send LL debugging messages to the modem. This can happen if
we compile the Samsung SoC with PM debugging as in andy's git branch.



From 452c17990dfeb9c2eb2ffa7ded4d24ddb1bcd4b2 Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Sun, 23 Mar 2008 00:57:28 +0100
Subject: [PATCH] Never ever send debug messages to the modem. The serial
console is
 on UART2, send the messages for decompressing  and low level debugging
 there.

Signed-Off-by: Holger Freyther <zecke@openmoko.org>
---
 defconfig-2.6.24 |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/defconfig-2.6.24 b/defconfig-2.6.24
index e57081a..cce3737 100644
--- a/defconfig-2.6.24
+++ b/defconfig-2.6.24
@@ -163,7 +163,7 @@ CONFIG_CPU_LLSERIAL_S3C2440=y
 #
 CONFIG_S3C2410_PM_DEBUG=y
 # CONFIG_S3C2410_PM_CHECK is not set
-CONFIG_S3C_LOWLEVEL_UART_PORT=0
+CONFIG_S3C_LOWLEVEL_UART_PORT=2
 
 #
 # S3C2400 Machines
@@ -1756,7 +1756,7 @@ CONFIG_DEBUG_ERRORS=y
 CONFIG_DEBUG_LL=y
 # CONFIG_DEBUG_ICEDCC is not set
 # CONFIG_DEBUG_S3C_PORT is not set
-CONFIG_DEBUG_S3C_UART=0
+CONFIG_DEBUG_S3C_UART=2
 
 #
 # Security options
-- 
1.5.6.3

